render-utils.js

```diff
@@ -0,0 +1,13 @@
+export function renderShoppingList(list) {
+    const div = document.createElement('div');
+    const p = document.createElement('p');
+
+    div.classList.add(list.complete ? 'complete' : 'incomplete');
```

Comment:
You will use this is the update a item maybe a button to toggle style on the display